webdriver/tests/support/fixtures.py

```diff
@@ -24,6 +25,8 @@ def pytest_configure(config):
     # register the capabilities marker
     config.addinivalue_line("markers",
         "capabilities: mark test to use capabilities")
+    config.addinivalue_line("markers",
+        "bidi: mark test to use bidi")
```

Comment:
Instead of a marker, which would mean that we have to add it before each and every test method, having a dedicated `bidi_session` fixture would cause way lesser code to write:

```
async def test_log_events(bidi_session, url):
    bidi_session.url = url("/tests/bidi/support/log_events.html")
    ...
```

That way we can also factor out all the bidi specific code from the `session` fixture itself, and have a clear separation for the BiDi extension.

What do you think?

Reply:
I think that's a good idea. It increased some code in the fixture by not reusing the session constructor but would reduce markers as number of bidi tests increases.

I created a new current_bidi_session global variable in the new commit and modified the tests to use the new bidi_session fixture. I think this is a big change compare to previous changes. I would consider reverting the change and merge what we have so far and create a separate issue to handle it in case there is unintended effect.

Reply:
Would be fine with me to move it to a follow-up issue/PR.

Reply:
Thanks, reverted the latest commit.